website/source/docs/providers/google/r/compute_instance_group_manager.html.markdown

```diff
@@ -1,29 +1,29 @@
 ---
 layout: "google"
-page_title: "Google: google_compute_instance_group_manager"
-sidebar_current: "docs-google-compute-instance-group-manager"
+page_title: "Google: google_compute_region_instance_group_manager"
+sidebar_current: "docs-google-compute-region-instance-group-manager"
```

Comment:
This looks like you just updated the `compute_instance_group_manager` docs instead of copying them and adding a new resource.